drivers/soundwire/stream.c

```diff
@@ -587,10 +587,11 @@ static int sdw_notify_config(struct sdw_master_runtime *m_rt)
 
 		if (slave->ops->bus_config) {
 			ret = slave->ops->bus_config(slave, &bus->params);
-			if (ret < 0)
+			if (ret < 0) {
 				dev_err(bus->dev, "Notify Slave: %d failed\n",
 					slave->dev_num);
-			return ret;
+				return ret;
+			}
```

Comment:
@RanderWang I think the commit message does not describe what this change is.
What is happening with this patch is that you will go through all slaves instead of returning after the first one.

So the commit message should be

soundwire: stream: fix support for multiple Slaves on the same link

The existing code will unconditionally return after dealing with the first Slave on a link. This return should only happen when there is an error case.